packages/Table/src/Table.styles.js

```diff
@@ -59,6 +59,8 @@ export const TD = styled.td(({ borderType }) => {
 export const TH = styled.th(({ borderType }) => {
   return css`
     ${borderType in borderTypesStyles ? borderTypesStyles[borderType] : ""}
+    font-size: 16px;
+    font-weight: 600;
```

Comment:
Do we use font-weight 600? I usually see 500/700, or just bold

Reply:
Datagrid seems to use `600` so I was following suit, but `bold` makes sense to me as well.